readme.md

```diff
@@ -306,6 +306,8 @@
 | [scienceimage](https://www.scienceimage.csiro.au) | An image library specializing in science and nature images |
 | [Integration & Application Network Image Library](https://ian.umces.edu/imagelibrary) | Free images to provide scientists, resource managers, government agencies, community groups and graphics professionals with a resource for enhancing science communication. |
 | [Saxifraga](http://www.freenatureimages.eu) | Free nature images |
+| [PngPix](http://www.pngpix.com/) | Free Transparent PNG Images For Everyone |
```

Comment:
No license on the website. Are you sure its 100% free? also, looks like you need a copyright for the images to be used.